apps/meteor/tests/data/permissions.helper.ts

```diff
@@ -0,0 +1,66 @@
+import { ISetting } from '@rocket.chat/core-typings';
+import { IS_EE } from '../e2e/config/constants';
+import { api, credentials, request } from './api-data';
+
+export const updatePermission = (permission:string, roles:string[]):Promise<void|Error> =>
+	new Promise((resolve,reject) => {
+		request
+			.post(api('permissions.update'))
+			.set(credentials)
+			.send({ permissions: [{ _id: permission, roles }] })
+			.expect('Content-Type', 'application/json')
+			.expect(200)
+			.end((err?:Error) => setTimeout(() => !err && resolve() || reject(err), 100));
+	});
+
+export const updateEEPermission = (permission:string, roles:string[]):Promise<void|Error> =>
+	IS_EE ? new Promise((resolve,reject) => {
+		request
+			.post(api('permissions.update'))
+			.set(credentials)
+			.send({ permissions: [{ _id: permission, roles }] })
+			.expect('Content-Type', 'application/json')
+			.expect(200)
+			.end((err?:Error) => setTimeout(() => !err && resolve() || reject(err), 100));
+	}) : Promise.resolve();
```

Comment:
Can we change `updateEEPermission` to just call updatePermission and not duplicate the call?
something like 
```
updateEEPermission => IS_EE ? updatePermission(fasdfasdf) : Promise.resolve();
```

Reply:
Nice idea :rocket: